app/assets/javascripts/components/physical_infrastructures/physical-server-toolbar.js

```diff
@@ -0,0 +1,50 @@
+ManageIQ.angular.app.component('physicalServerToolbar', {
+  bindings: {
+    physicalServerId: '=?',
+  },
+  controllerAs: 'vm',
```

Comment:
`vm` is the standard abbreviation for `view model` (I believe).
In the angular community, it is almost always used in the context of `controllerAs`.

I suppose we could call it whatever we want.
Though, I just don't know if we should.